src/components/form/signup.js

```diff
@@ -58,86 +89,232 @@ const LoginLink = styled.a`
     cursor: pointer;
 `
 
+const validation_message = {
```

Comment:
Maybe we could remove a couple of line by removing the nested `email` key-value pair and change the variable name to something like `const email_validation_message`? I am assuming that this is done because we might need to add message for other things like password validations. However, I feel that it is best practice to avoid writing code until the moment we need it, otherwise it will leave some unused fluff and make it harder to read.